frontend/pages/search.tsx

```diff
@@ -21,6 +21,10 @@ import {
   useState,
 } from "react";
 
+const DynamicSearchForm = dynamic(() => import("@/components/search/SearchForm"), {
+  ssr: false,
+});
```

Comment:
This drops the initial bundle size for the search page a good bit. This is good when a valid search page is directly accessed, since the form is hidden behind the sliding drawer, but this also makes the search form not render immediately when invalid search pages are directly accessed.